includes/widgets/playing-cards.php

```diff
@@ -0,0 +1,290 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor HTML widget.
+ *
+ * Elementor widget that insert a custom HTML code into the page.
```

Comment:
fic the comment :)

Reply:
Can you compare to other comments I think the extra line in the comment is the main issue.